### ISSUE TITLE
[BUG] Legacy context API has been detected within a strict-mode tree:      in StrictMode

### ISSUE THREAD

**Issue by greena13** [greena13/react-hotkeys#151]
**Describe the bug**

It seems a regression in the compliance with the new context API was introduced with `GlobalHotKeys`.

```
vendors~main.64d108ae760cbd63d1ea.bundle.js:138014 Warning: Legacy context API has been detected within a strict-mode tree: 
    in StrictMode

Please update the following components: GlobalHotKeys

Learn more about this warning here:
https://fb.me/react-strict-mode-warnings
```

**Comment by greena13** [comment]
Resolved by #176

**Comment by greena13** [comment]
Fix is now available in [v2.0.0-pre6](https://github.com/greena13/react-hotkeys/releases/tag/v2.0.0-pre6).
